salt/metalk8s/repo/redhat.sls

```diff
@@ -65,6 +65,9 @@ Configure {{ repo_name }} repository:
   {%- endif %}
     - repo_gpg_check: {{ repo_config.repo_gpg_check }}
     - enabled: {{ repo_config.enabled }}
+    # Set priority to 1, even if the `yum-plugin-priorities` is not installed,
+    # so that our repos has priority over others
+    - priority: 1
```

Comment:
Lower numbers mean higher priority?

Reply:
Yep and if for example epel has priority=5 and our one has priority=4 you cannot install salt 3000 at all :) (except if you disable the priority plugin)

Reply:
? So it only works with `priority=1`? I don't understand

Reply:
No just you need higher priority than others repos that provide the same package or your package will be ignored 
(epel priority=4 provide salt-2015 so if our repo has priority=5 salt from our repo will just be ignored )
```
[root@test-salt-install ~]# yum install salt-minion-3000.3
Loaded plugins: fastestmirror, priorities
Loading mirror speeds from cached hostfile
 * base: mirroir.wptheme.fr
 * epel: epel.mirror.wearetriple.com
 * extras: centos.mirrors.proxad.net
 * updates: mirroir.wptheme.fr
236 packages excluded due to repository priority protections
No package salt-minion-3000.3 available.
Error: Nothing to do
```

Reply:
Oh OK, you switched them in the first version of your comment so I thought there was a weird behaviour. OK makes sense!